Tests/ValidationTests.swift

```diff
@@ -756,18 +756,19 @@ class AutomaticValidationTestCase: BaseTestCase {
 
 private enum ValidationError: Error {
     case missingData, missingFile, fileReadFailed
+    var error: AFError { return .responseValidationFailed(reason: .customValidationFailed(error: self)) }
 }
 
 extension DataRequest {
     func validateDataExists() -> Self {
         return validate { request, response, data in
-            guard data != nil else { return .failure(ValidationError.missingData) }
+            guard data != nil else { return .failure(ValidationError.missingData.error) }
```

Comment:
Users shouldn't have to do this, so perhaps `ValidationResult` should be left as `Result<Void, Error>`.

Reply:
Reverted.